hack/testing/check-EFK-running.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+TEST_DIVIDER="-------------------------------------------------------"
+NEEDED_COMPONENTS=("logging-es" "logging-kibana" "logging-fluentd")
+NEEDED_STREAMS=("logging-fluentd" "logging-elasticsearch" "logging-auth-proxy" "logging-kibana")
+COMPONENTS_COUNT=${#NEEDED_COMPONENTS[@]}
+STREAMS_COUNT=${#NEEDED_STREAMS[@]}
+
+echo "Checking component installation and if pods are running:"
+# Check that we have all IS
+echo $TEST_DIVIDER
+
+FOUND_STREAMS=(`oc get is | grep 'logging-' | cut -d" " -f 1`)
+IS_COUNT=${#FOUND_STREAMS[@]}
+IS_MESSAGE="[$IS_COUNT/$STREAMS_COUNT] image streams found."
+
+if [[ $IS_COUNT -ne $STREAMS_COUNT ]]; then
+  echo "Error - $IS_MESSAGE"
+
+  # check for elasticsearch, kibana, auth-proxy, and kibana images
+  for stream in "${NEEDED_STREAMS[@]}"; do
```

Comment:
Maybe early optimization, but you have some repeated logic here it looks like that is used below and could maybe be moved to a function.  Might also help clarify what is going on in the test.  Also, depending what @mwringe is doing for metrics e2e, maybe an opportunity for reuse.